examples/basic_timelock_ibe/backend/src/types.rs

```diff
@@ -106,13 +106,6 @@ impl Storable for LotInformation {
     const BOUND: Bound = Bound::Unbounded;
 }
 
-#[derive(CandidType, Deserialize, Clone, Debug)]
-pub struct SendMessageRequest {
-    pub receiver: Principal,
-    #[serde(with = "serde_bytes")]
-    pub encrypted_message: Vec<u8>,
-}
-
 #[derive(CandidType, Deserialize, Clone, Debug)]
 pub struct GetEncryptedIbeKeyRequest {
```

Comment:
Isn't this `GetEncryptedIbeKeyRequest` also unused? Both here in `basic_timelock_ibe` and in `basic_ibe`.